src/explore-education-statistics-admin/src/services/releaseAncillaryFileService.ts

```diff
@@ -28,6 +28,12 @@ export interface UploadAncillaryFileRequest {
   file: File;
 }
 
+export interface ReleaseAncillaryFileUpdateRequest {
+  title: string;
+  summary: string;
+  file?: File | null;
+}
```

Comment:
We don't need this. We can just update `AncillaryFileUpdateRequest` to have the `file` property.